src/test/java/de/medizininformatikinitiative/flare/service/DataStoreIT.java

```diff
@@ -102,6 +102,27 @@ void execute_twoObservationsFromTwoPatients() {
         StepVerifier.create(result).expectNext(Population.of("0", "1").withCreated(FIXED_INSTANT)).verifyComplete();
     }
 
+    @Test
+    void execute_OneObsWithoutReference_FromOnePat(){
+        createObservation_withoutReference();
+
+        var result = dataStore.execute(Query.ofType("Observation"));
+
+        StepVerifier.create(result).expectNext(Population.of().withCreated(FIXED_INSTANT)).verifyComplete();
+    }
+
+    @Test
+    @DisplayName("A Resource without a Reference to a Patient can't be used")
```

Comment:
Please give both tests a display name and make clear that the first is about one invalid reference and the second with one invalid and one valid reference.